### ISSUE TITLE
Build: Links used instead of buttons

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#75]
URL: https://hcdev.openplus.ca/fr/node/2525/build#Section1 (click on Build for any Recall or Alert from Dashboard)
User Agent: Chrome, Firefox, NVDA
Language: Both

Issue Description: A few buttons are links but made to look like buttons. Links usually take user to another page or a new section but button triggers an action. These seem to be doing an action like locking to edit, or editing a section. These are also referred as "buttons" in the help manual. Please see screenshots. 

![image](https://user-images.githubusercontent.com/14280062/129220745-330873c7-385f-4921-98a7-6be888e26f82.png)

![image](https://user-images.githubusercontent.com/14280062/129220680-0afd2bc9-60b4-4b4c-b034-ddc727e80e68.png)

Steps to reproduce: Go to one of the Build pages and listen with the screen reader. Also inspect code.

Expected results: Since these perform an action like in a form, they should be made a button. Suggestion would be to add role="button" in the tag.

**Comment by hydegeoff** [comment]
All these items are not identified as buttons when using NVDA.  In the screenshot, I have identified elements that should identified as buttons in FR/EN.  

![Items are identified as links when they should be buttons](https://user-images.githubusercontent.com/88147013/160837829-ed5027cb-9304-4635-a60d-93f8ee4f2f95.jpg)

**Comment by echouin** [comment]
I believe this issue was resolved.
